drushrc.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+$options['uri'] = 'http://localhost:8888';
```

Comment:
For some reason the drush rc file isn't doing what it should, You can use the drush.yml file in the drush folder in Your project root instead and You can have the following inside it:

`options:
  uri: 'http://localhost:8888'`

once You've done that verify it works and delete this file.

Reply:
done.